src/server/worker/master.go

```diff
@@ -249,6 +249,16 @@ func (a *APIServer) spoutSpawner(pachClient *client.APIClient) error {
 	}
 	puller := filesync.NewPuller()
 
+	if a.pipelineInfo.Spout.Marker != "" {
+		markerBranch, err := pachClient.InspectBranch(a.pipelineInfo.Pipeline.Name, "marker")
```

Comment:
I'd have `"marker"` defined as a constant somewhere so we can easily change it if need be.